diff --git a/src/run.c b/src/run.c
index e86b52e..a554f5e 100644
--- a/src/run.c
+++ b/src/run.c
@@ -125,6 +125,7 @@ crun_command_run (struct crun_global_arguments *global_args, int argc, char **ar
   cleanup_free char *config_file_cleanup = NULL;
 
   crun_context.preserve_fds = 0;
+  crun_context.no_new_keyring = true;
 
   argp_parse (&run_argp, argc, argv, ARGP_IN_ORDER, &first_arg, &crun_context);
   crun_assert_n_args (argc - first_arg, 1, 1);
